### ISSUE TITLE
Disable exception canvas display after dweet is posted

### ISSUE THREAD

**Issue by mreinhardt** [lionleaf/dwitter#147]
The new exception canvas display 8305d6a is very useful during development, but can it be disabled after a dweet is posted? Sometimes you may have an exception that doesn't interfere with functionality because it only occurs on the first iteration before vars are initialized. Since we're golfing this is sometimes a necessity.

See: https://www.dwitter.net/d/1521

**Comment by lionleaf** [comment]
I agree. Just remember to switch it on once you start changing someone elses dweet.

Another example: https://www.dwitter.net/d/611

**Comment by iver56** [comment]
Same thing occurs here:
https://www.dwitter.net/d/1230

**Comment by mreinhardt** [comment]
@lionleaf maybe the same code that causes the "post remix" button to appear could trigger re-enabling it also.

**Comment by lionleaf** [comment]
This would also solve certain issues with the exception area being used to escape the iframe sandbox, for instance by throwing an error with hundreds of newline characters.

It would however break some of the dweets that actively use `throw t` for instance as a short "animated" dweet
